components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConsumerImpl.java

```diff
@@ -5461,7 +5461,7 @@ private String getOpenAPIDefinitionForDeployment(Identifier apiId, String synaps
             api.setUriTemplates(oasParser.getURITemplates(definition));
             apiTenantDomain = MultitenantUtils.getTenantDomain(
                     APIUtil.replaceEmailDomainBack(api.getId().getProviderName()));
-            if (synapseEnvName != null) {
+            if (!StringUtils.isEmpty(synapseEnvName)) {
```

Comment:
This is fine, but there is also "StringUtils.isNotEmpty" available, IINM. Just  FYI.